e2e/testsuite/testsuite.go

```diff
@@ -286,6 +287,14 @@ func (s *E2ETestSuite) RecoverRelayerWallets(ctx context.Context, relayer ibc.Re
 	return nil
 }
 
+// Transfer broadcasts a MsgTransfer message.
+func (s *E2ETestSuite) Transfer(ctx context.Context, chain *cosmos.CosmosChain, user *ibc.Wallet,
```

Comment:
we now have multiple tests requiring the `Transfer` function so I moved it to here. There may be a better way, I'm open to suggestions!

Reply:
It's not possible to import types from `*_test.go` files. So the test suites can only be shared within the packages.

A possible idea (for a different PR) could be to try an extract out these functions to package level functions. E.g. `transfer.PerformTransfer(...)` or similar, and not have these as methods on a test suite at all.